mock/service/service.go

```diff
@@ -73,6 +119,33 @@ func New(config Config) Service {
 		nodeID: config.DriverName,
 		config: config,
 	}
+	if config.ExecHooks != nil {
+		s.hooksVm = otto.New()
+		s.hooksVm.Run(`__grpcCode = {
+			OK: 0,
+			CANCELED: 1,
+			UNKNOWN: 2,
+			INVALIDARGUMENT: 3,
+			DEADLINEEXCEEDED: 4,
+			NOTFOUND: 5,
+			ALREADYEXISTS: 6,
+			PERMISSIONDENIED: 7,
+			RESOURCEEXHAUSTED: 8,
+			FAILEDPRECONDITION: 9,
+			ABORTED: 10,
+			OUTOFRANGE: 11,
+			UNIMPLEMENTED: 12,
+			INTERNAL: 13,
+			UNAVAILABLE: 14,
+			DATALOSS: 15,
+			UNAUTHENTICATED: 16
+		};`)
+
+		_, err := s.hooksVm.Run(s.config.ExecHooks.Globals)
+		if err != nil {
+			fmt.Printf("Warning: Error encountered in the global exec hook: %v\n", err)
```

Comment:
I would prefer `os.Exit`, otherwise the driver may do something unexpected in its CSI calls.

Reply:
Done.